presentation/src/main/java/com/open/presentation/ui/screens/WeatherScreen.kt

```diff
@@ -78,7 +84,17 @@ fun WeatherScreen(
             }
 
             is UiState.Empty -> {
-                Text(text = "Enter a city to get weather information")
+                LaunchedEffect(Unit) {
+                    viewModel.viewModelScope.launch {
+
+                    }
+
+                }
+                val name = viewModel.nameFlow.collectAsState("").value
+                val temp = viewModel.temp.collectAsState("").value
```

Comment:
Do these states need to be inside the viewModel? Can we not have them be local rememberSaveable states?

Reply:
as these states are fetch  from ViewModel that is why using from viewmodel